lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -54,18 +57,21 @@
   private final REEFFileNames fileNames;
   private final YarnConfiguration yarnConfiguration;
   private final ClasspathProvider classpath;
+  private final Set<ConfigurationProvider> configurationProviders;
 
   @Inject
   YarnJobSubmissionClient(final JobUploader uploader,
                           final YarnConfiguration yarnConfiguration,
                           final ConfigurationSerializer configurationSerializer,
                           final REEFFileNames fileNames,
-                          final ClasspathProvider classpath) {
+                          final ClasspathProvider classpath,
+                          final @Parameter(DriverConfigurationProviders.class) Set<ConfigurationProvider> configurationProviders) {
```

Comment:
Why do we need these here? Wouldn't the config be merged in in `REEFImplementation` anyhow?